flow/scripts/variables.yaml

```diff
@@ -379,6 +379,12 @@ DPO_MAX_DISPLACEMENT:
   description: >
     Specifies how far an instance can be moved when optimizing.
   default: 5 1
+GPL_ALLOW_REVERT_IF_DIVERGE:
+  description: >
+    Enable reverting to saved snapshot if a divergence is detected.
```

Comment:
this could do with a bit of elaboration. When would the user use this?

Also, isn't this a stopgap feature? Worth mentioning.